framework/test/unit/modules/auth/module.spec.ts

```diff
@@ -30,75 +27,32 @@ import {
 	authAccountSchema,
 	AuthAccountStore,
 } from '../../../../src/modules/auth/stores/auth_account';
+import {
+	chainID,
+	registerMultisigTx,
+	multisigAddress,
+	keyPairs,
+	multisigParams,
+} from './multisig_fixture';
 
 describe('AuthModule', () => {
-	let decodedMultiSignature: any;
-	let validTestTransaction: any;
 	let stateStore: any;
 	let authModule: AuthModule;
-	let decodedBaseTransaction: any;
-	let passphrase: any;
-	let passphraseDerivedKeys: any;
-	let senderAccount: any;
-
-	const { cloneDeep } = ObjectUtils;
-	const subStoreMock = jest.fn();
-	const storeMock = jest.fn().mockReturnValue({ getWithSchema: subStoreMock });
 
-	const defaultTestCase = fixtures.testCases[0];
-	const chainID = Buffer.from(defaultTestCase.input.chainID, 'hex');
+	const authAccountStoreMock = jest.fn();
+	const storeMock = jest.fn().mockReturnValue({ getWithSchema: authAccountStoreMock });
```

Comment:
stub should be created in the `beforeEach` to keep each test atomic

Reply:
In this specific test suite, these mock values are not modifed during the execution of the test. Also, they were already outside of `beforeEach` even before the changes introduced by this PR. I just gave variables a more accurate name, so Git diff shows them as modified lines.

Nevertheless, I do agree that as in general it is better to re-initialize the stores before each test case 👍🏻